build.macos32x86/common/Makefile.plugin

```diff
@@ -184,6 +196,21 @@ $(PLUGINPLIST): $(PLATDIR)/iOS/plugins/Info.plist getversion
 	sed "s/\$$(VERSION)/`./getversion VERSION_TAG`/" $< | \
 	sed "s/\$(LIBNAME)/$(LIBNAME)/" > $(PLUGINPLIST)
 
+$(PLUGINDYLIB): $(PLUGINREQS) $(VMDIR) $(OBJDIR) $(LIBOBJ)
+ifneq ($(LINK_WITH_CPP),)
+	$(LDCXX) \
+	$(DYFLAGS) \
+	-install_name @executable_path/Plugins/$(LIBNAME).dylib \
```

Comment:
I tried a lot with rpath and I was not able to solve it :(
a bit of help here would be very appreciated :)